samples/hello++/Makefile

```diff
@@ -0,0 +1,4 @@
+XBE_TITLE=hello++
+SRCS = $(wildcard $(CURDIR)/*.cpp)
```

Comment:
Consider listing source files explicitly; we should do this in all our samples.

Wildcards are error prone.

Reply:
It also may be worth including `GEN_XISO = $(XBE_TITLE).iso` in this sample as well as some of the others that are missing it.